tasmota/xdrv_12_home_assistant.ino

```diff
@@ -232,10 +232,10 @@ void HassDiscoverMessage(void) {
                    SettingsText(SET_MQTTPREFIX2),
                    SettingsText(SET_MQTTPREFIX3));
 
+#ifdef USE_LIGHT
   uint8_t light_idx = MAX_RELAYS + 1;                          // Will store the starting position of the lights
```

Comment:
Don't change this. As said before these values are needed for discovery.

Reply:
Should be resolved.